GraphAlgorithms/Program.cs

```diff
@@ -27,10 +28,35 @@ private static void Main(string[] args)
         }, nameOption, algorithmTypeOption);
         rootCommand.Add(dimacsCommand);
 
-        var cliqueCommand = new Command("clique", "Find clique");
+        var cliqueCommand = new Command("max-clique", "Find maximum clique");
+        var fileOption = new Option<string>(name: "--file",
+            description: "Path to file with graphs");
+        var indexOption = new Option<int>(name: "--index",
```

Comment:
all of this stuff should be required var indexOption = new Option<int>(name: "--index",
            description: "Index of the graph in the file",
            getDefaultValue: () => 0)
        {
            IsRequired = true
        };

Reply:
added required whenever there's no default.
also added aliases because this is becoming lengthy